index.js

```diff
@@ -45,17 +53,35 @@ exports.handlers = {
         recoverable: true
       }).tags
 
+      let style = MERMAID_CONFIG.style ? ' style="' + MERMAID_CONFIG.style + '"' : ''
```

Comment:
@cmidgley this is not-blocking, but we should prolly `escapeHTML()` the `style` value too. 
As currently built, if we wanna use a quote in the style, this would break the html string. e.g. `"style": "background: url(\"this_would_break.jpeg\")"`